godtools/App/Features/Dashboard/Presentation/ChooseYourOwnAdventure/ChooseYourOwnAdventureViewModel.swift

```diff
@@ -59,93 +59,112 @@ class ChooseYourOwnAdventureViewModel: MobileContentPagesViewModel {
             hidesBackButton = false
         }
     }
-}
-
-// MARK: - Inputs
-
-extension ChooseYourOwnAdventureViewModel {
     
-    @objc func homeTapped() {
-        flowDelegate?.navigate(step: FlowStep.backTappedFromChooseYourOwnAdventure)
+    // MARK: - Page Navigation
+    
+    override func getInitialPages(pageRenderer: MobileContentPageRenderer) -> [Page] {
+            
+        let allPages: [Page] = pageRenderer.getAllPageModels()
+        
+        if let introPage = allPages.first(where: {$0.id == "intro"}) {
+            return [introPage]
+        }
+        else if let firstPage = allPages.first {
+            return [firstPage]
+        }
```

Comment:
this logic should be the first page that isn't hidden

Reply:
Good call thanks @frett.